### PR TITLE
Collection

### PR DESCRIPTION
**Pull request by danacebryantalfonso**

### CODE REVIEW

File: Collection/src/ListArray.kt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    println(colors.size)
+
+    ArrayListSample()
+
+    exercise1()
+    exercise2()
+}
+
+fun ArrayListSample()
```
Comment (rexendy-lig) [inline Collection/src/ListArray.kt:26]: function name should be camelCase

File: Collection/src/ListArray.kt [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+{
+    val colors: ArrayList<String> = arrayListOf("blue", "red")
```
Comment (rexendy-lig) [inline Collection/src/ListArray.kt:28]: val colors = arrayListOf("blue", "red")

File: Collections/src/SetAndHashSet.kt [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        val numbers = setOf(6,37,6,42)
+        println(numbers)
+
+        val numbers2 = setOf<Int>()
+        println(numbers2)
+
+        val numbers3 = setOf(6, 5, 2, null, 8, 5, 7, null, 4, null)
+        println(numbers3)
+    }
+    fun hashSet() {
```
Comment (rexendy-lig) [inline Collections/src/SetAndHashSet.kt:20]: pls add space

**Comment by rexendy-lig** [comment]
@danacebryantalfonso , pls add necessary files/folders to .gitignore

make this as your ref: https://github.com/github/gitignore/blob/master/Android.gitignore

thank you

**Review by rexendy-lig — CHANGES_REQUESTED** [review]

File: Collection/src/ListArray.kt (hunk 2 above)
Comment (danacebryantalfonso) [reply Collection/src/ListArray.kt:28]: Sorry for this. I refactored the folder and forgot to remove it. This is fixed in this commit ed087a565afad57e315d26c49996ef78139bb607.

File: Collections/src/SetAndHashSet.kt (hunk 3 above)
Comment (danacebryantalfonso) [reply Collections/src/SetAndHashSet.kt:20]: changed in ed087a565afad57e315d26c49996ef78139bb607

File: Collection/src/ListArray.kt (hunk 1 above)
Comment (danacebryantalfonso) [reply Collection/src/ListArray.kt:26]: this file has been removed in 9617ad04fa7a33c400c3699514db9233f15c3451

**Comment by danacebryantalfonso** [comment]
> @danacebryantalfonso , pls add necessary files/folders to .gitignore
> 
> make this as your ref: https://github.com/github/gitignore/blob/master/Android.gitignore
> 
> thank you

Added some necessary files and folders in .gitignore 2c1e831ce9d392ab852e7d503c9e15321aec7e37. Some are just for the future android development.

**Pull request merged**